server/models/parties.js

```diff
@@ -0,0 +1,28 @@
+const partiesModel = [
+  {
```

Comment:
Why partiesModel? Let your variable names say exactly what it represents. Is this a list of party model or a list of parties?